src/IO/Resources/ClilocLoader.cs

```diff
@@ -90,6 +90,35 @@ public override Task Load()
                         return;
                     }
 
+                    if (_cliloc != "Cliloc.enu")
```

Comment:
Because CUO runs in all OS we have to ignore capitalization, so i suggest to replace this IF statement with:

```csharp

if (string.Compare(_cliloc, "cliloc.enu", StringComparison.InvariantCultureIgnoreCase) == 0)
```

Reply:
Sorry i was wrong: the comparisoon have to be != 0

Reply:
sorry I am new to git, you can change it?

Reply:
just apply the modify and push it